lib/export-types.ts

```diff
@@ -1,4 +1,5 @@
 export * from './common-types'
+export * from './plain/common-types'
```

Comment:
export all or just that specific type?  Atm it's the same, but in the future we might export more types from `common-types`

Reply:
All of them. Hence the name 'common-types'